geniza/corpus/tests/test_corpus_models.py

```diff
@@ -1121,3 +1122,16 @@ def test_thumbnail(self):
         block = TextBlock.objects.create(document=doc, fragment=frag, side="r")
         with patch.object(frag, "iiif_thumbnails") as mock_frag_thumbnails:
             assert block.thumbnail() == mock_frag_thumbnails.return_value
+
+
+@pytest.mark.django_db
+class TestDocumentSignalHandlers:
+    # Other tests are found in `test_corpus_signals.py`
+
+    def test_items_to_index(document, footnote):
+        """Test that prefetching is properly configured."""
+        # Because of lazy loading, querysets must be executed to test prefetches.
+        # Footnote fixture must be included to check source/creator prefetching.
+        docs = Document.items_to_index()
+        assert docs
+        assert type(docs) is MultilingualQuerySet
```

Comment:
this method isn't part of the signal handlers, it's on `Document`; but easiest to test as a standalone method

```suggestion
@pytest.mark.django_db
def test_items_to_index(document, footnote):
    """Test that prefetching is properly configured."""
    # Because of lazy loading, querysets must be executed to test prefetches.
    # Footnote fixture must be included to check source/creator prefetching.
    docs = Document.items_to_index()
    assert docs
    assert type(docs) is MultilingualQuerySet
```

Reply:
Wow yeah, I just had it in my head that it was in the signal handlers class, thanks